amundsen_application/static/js/index.tsx

```diff
@@ -17,7 +17,8 @@ import NavBar from './components/NavBar';
 import NotFoundPage from './components/NotFoundPage';
 import Preloader from "components/common/Preloader";
 import ProfilePage from './components/ProfilePage';
-import SearchPage from './components/SearchPage';
+import SearchPage from './components/SearchPage'; 
+import HomePage from './components/HomePage'
```

Comment:
nit: place `HomePage` higher up in alphabetical order, and looks like I missed this file when we switched to absolute imports. We can remove the relative imports in this file and switch to:
`import Foo from 'components/Foo'` and `import Thing from 'ducks/pathToThing'`